src/main.rs

```diff
@@ -230,6 +230,10 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
             copper_to_string(profitable_item.profit.to_integer()),
             profitable_item.profit_per_crafting_step().to_integer()
         );
+        println!(
+            "Min sell price per unit: {}",
```

Comment:
What do you think of calling this the "break-even price"?

Reply:
Added the break-even price, as well as a max price so I output the sell range.